resources/sql/application-queries.sql

```diff
@@ -377,6 +377,19 @@ FROM applications a
   JOIN latest_version lv ON a.created_time = lv.latest_time
   JOIN forms f ON f.id = a.form_id;
 
+-- name: yesql-organization-oids-of-applications-of-persons
+-- Get the organization oids of the related forms
+
+WITH latest_version AS (
+    SELECT max(created_time) AS latest_time
```

Comment:
I think this max(created_time) has been used in all such cases, but wouldn't max(id) be simpler and guaranteed to be unique? Probably not worth changing though..